protocol/x/clob/keeper/stateful_order_state.go

```diff
@@ -59,11 +59,11 @@ func (k Keeper) SetLongTermOrderPlacement(
 	memStore.Set(orderIdBytes, longTermOrderPlacementBytes)
 
 	if !found {
-		// Increment the `to be committed` stateful order count.
-		k.SetToBeCommittedStatefulOrderCount(
+		// Increment the stateful order count.
+		k.SetStatefulOrderCount(
```

Comment:
This doesn't get called when the memclob is hydrated on app startup right?

Should we add logic in [this function](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/x/clob/keeper/orders.go#L1073) to set the memstore count properly?